docs/contributing/browser-support-for-contributors.stories.mdx

```diff
@@ -0,0 +1,23 @@
+import { Meta } from '../../.storybook/components';
+
+<Meta title="Introduction/Contributing/Browser support (for contributors)" />
+
+# Browser support (for contributors)
```

Comment:
Would love some feedback on this 🙌 I'm trying to document some of the quirks about browser support I've been running into when working with Circuit UI (and TypeScript)

Reply:
I love it!